components/config-mgmt-service/grpcserver/nodes.go

```diff
@@ -48,8 +48,10 @@ func (s *CfgMgmtServer) GetRuns(
 	page, pageSize := request.GetPagination().GetParameters()
 
 	// Date Range
-	if !params.ValidateDateRange(request.GetStart(), request.GetEnd()) {
-		return runs, status.Errorf(codes.InvalidArgument, "Invalid start/end time. (format: YYYY-MM-DD)")
+	if !params.ValidateDateTimeRange(request.GetStart(), request.GetEnd()) {
+		if !params.ValidateDateRange(request.GetStart(), request.GetEnd()) {
```

Comment:
Add this into one if statement only.

Reply:
fc279f928704bbf60776a53b815a3e2803ea5cfe both requested changes added.